pyproject.toml

```diff
@@ -35,9 +35,6 @@ documentation = "https://docs.saleor.io/"
   django-countries = "^7.2"
   django-filter = "^23.1"
   django-measurement = "^3.0"
-  # Version 0.18 introduces changes in index definitions.
-  # Upgrading to this version requires new migrations, which are already included in later versions.
-  # Block upgrading to prevent potential migration conflicts.
   django-mptt = ">=0,<0.18"
```

Comment:
You can also pin this specific version like this:
```
django-mptt==0.17
```
With a comment why we pin it that could be enough.

Reply:
But still it can be changed by someone